liiatools/datasets/s903/lds_ssda903_clean/logger.py

```diff
@@ -180,6 +203,14 @@ def save_errors_la(stream, la_log_dir):
                 ) as f:
                     f.write(match_error)
                     f.write("\n")
+            column_error = getattr(event, "extra_column_error", None)
+            if column_error:
+                with open(
+                    f"{os.path.join(la_log_dir, event.filename)}_error_log_{start_time}.txt",
```

Comment:
FYI, the format I used above is ISO 8601 and is a general standard for dates in filenames.

Reply:
This part of the code is a copy of what we had for all the other logs, but if it needs fixing, then we should do that! I think we can choose to define it how we like. Should we implement this change using the standard format for start_time across all of the logs for the code?

Reply:
This was just something I noticed this time through. If it is easy to fix, we should do throughout to be consistent. Since spacing in filenames can cause problems in niche situations, then I suggest we at least think about transitioning to a more universal standard format across the code base, yes.  Do you see any complications changing this for the other logs?

Reply:
Yes, let's fix. Should we do this as part of this PR or as part of another one that applies across all of the processes?